### ISSUE TITLE
Videos created are sped up and out of sync FPS detection issuew

### ISSUE THREAD

**Issue by mewhoexactlywhat** [C0untFloyd/roop-unleashed#111]
**Describe the bug**
The output videos are sped up and the audio is our of sync.  this started happening after the recent update.
seems to be an issue with detecting the correct fps of the video...it detects video fps at 240 but video is  60fps causing all sorts of issues
see screenshot
<img width="960" alt="Screenshot 2023-08-15 at 10 07 40" src="https://github.com/C0untFloyd/roop-unleashed/assets/44648438/9465f268-5f66-4d0f-a609-00229bc1499a">

**To Reproduce**
running regularly
**Details**
What OS are you using?
- [ ] Linux
- [ ] Linux in WSL
- [ X] Windows 
- [ ] Mac

Are you using a GPU?
- [ ] No. CPU FTW
- [ X] NVIDIA
- [ ] AMD
- [ ] Intel
- [ ] Mac

**Which version of roop unleashed are you using?**
2.7.3

**Comment by mewhoexactlywhat** [comment]
seems to be an issue with the newer 6.0 version of ffmpeg.  issue was resolved when i rolled ffmpeg back to 5.1.2 version

**Comment by mewhoexactlywhat** [comment]
upon further testing, only the first swap in a batch job processes correctly...all others still have the issue of detecting incorrect fps

**Comment by C0untFloyd** [comment]
Must be something with ffmpeg's ffprobe. Nothing changed in the code since months and the default (if ffmpeg fails) is 30 fps.

**Comment by mewhoexactlywhat** [comment]
it happens only on videos above 30fps...unfortunately, i film everything in 60fps.  if i re-encode down to 30, it works however, i still can't run batch jobs...it just doesn't swap in the batch, just outputs the original file again...very strange.  i'm going to wipe and re-install and see what happens.

**Comment by phineas-pta** [comment]
try run this command
```
ffprobe -select_streams v:0 -show_entries stream=r_frame_rat -of default=noprint_wrappers=1:nokey=1 YOUR_VIDEO_FILE
```

if it still output 240fps, it means the way roop detect fps fail for your video

**Comment by C0untFloyd** [comment]
Did you try @phineas-pta suggestion?

**Comment by mewhoexactlywhat** [comment]
i did, and it shows incorrect frame rate...if i re-encode to constant bitrate, all is well...it must be something with how my camera is producing the files, but its very strange because other files created with the same camera before were working just fine, suddenly this issue came up....no matter, i just added re-encoding into the workflow and its fine.  thanks for all the help!

**Comment by TrickleDownClown** [comment]
I'm having the same issue and the suggestions didn't help me. I'll try reencoding as well.

**Comment by phineas-pta** [comment]
if u guys know any ffprobe/ffmpeg command which can detect correctly fps in your case, consider sharing it so other can test

otherwise it's video problem and nothing can do to improve roop

**Comment by Gasbeutel** [comment]
> seems to be an issue with the newer 6.0 version of ffmpeg. issue was resolved when i rolled ffmpeg back to 5.1.2 version

this worked for me
